modin/pandas/dataframe.py

```diff
@@ -98,14 +98,14 @@ def _build_repr_df(self, num_rows, num_cols):
         num_rows_for_head = num_rows // 2 + 1
         num_cols_for_front = num_cols // 2 + 1
 
-        if len(self.index) <= num_rows:
+        if len(self._query_compiler.index) <= num_rows:
```

Comment:
Why does this change? `self.index` is a property view on `self._query_compiler.index`.

Reply:
That change was just for debugging and was not supposed to be pushed, my bad!